build/build_openr.sh

```diff
@@ -286,6 +286,7 @@ install_openr() {
   cd "$BUILD_DIR/../openr/py"
   sudo pip install cffi
   sudo pip install future
+  sudo pip install --upgrade typing
```

Comment:
Can you add this to `setup.py` instead https://github.com/facebook/openr/blob/master/openr/py/setup.py#L65 as it is cleaner solution.